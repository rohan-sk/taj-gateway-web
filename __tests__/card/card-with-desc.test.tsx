import { render } from "@testing-library/react";
import dynamic from "next/dynamic";
const CardWithDesc = dynamic(
  () => import("../../components/card/card-with-desc.component")
)

it("shows card with description component", () => {
  const { asFragment } = render(<CardWithDesc />);
  expect(asFragment()).toMatchSnapshot();
});
