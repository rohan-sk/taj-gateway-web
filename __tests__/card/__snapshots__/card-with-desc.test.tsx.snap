// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shows card with description component 1`] = `
<DocumentFragment>
  <div
    class="MuiBox-root css-wlh1v7"
  >
    <div
      class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-m2jzmr-MuiPaper-root-MuiCard-root"
    >
      <div
        class="MuiBox-root css-xmkxhf"
      >
        <hr
          class="MuiDivider-root MuiDivider-fullWidth css-1ngjjm0-MuiDivider-root"
        />
        <div
          class="MuiBox-root css-0"
        >
          <p
            class="MuiTypography-root MuiTypography-body1 css-jyynly-MuiTypography-root"
          />
          <p
            class="MuiTypography-root MuiTypography-body1 css-1mraeni-MuiTypography-root"
          />
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
